operator/operatortest/envtest.go

```diff
@@ -181,6 +181,31 @@ func (ts *Suite) FetchResources(objectList client.ObjectList, opts ...client.Lis
 	ts.Require().NoError(ts.Client.List(ts.Context, objectList, opts...))
 }
 
+// ExistsResource checks if the given resource exists.
+// Test fails for any error.
+func (ts *Suite) ExistsResource(name types.NamespacedName, object client.Object) {
+	ts.Require().NoError(ts.Client.Get(ts.Context, name, object))
+}
```

Comment:
I would argue again that having a precise intention with a function even if the implementation is the same as to another function is ok in testing, the whole point which I see in testing is to make it as readable as possible even if I have the same code written twice. Also we could call the `FetchResource` inside `ExistsResource`. I thought to remove this function not because it is the same to `FetchResource` but because ultimately I haven't used it in any tests.

Reply:
I don't like the idea to introduce functions that are literal duplications just with a different name just to rephrase the intent...